app/main/controllers/activity/RTMediaBuddyPressActivity.php

```diff
@@ -76,6 +78,25 @@ public function __construct() {
 		}
 	}
 
+	/**
+	 * Adds rtmedia_update in editable activity types when BuddyPress Edit Activity plugin is active.
+	 *
+	 * @param array $option Editable activity types.
+	 *
+	 * @return array Modified editable activity types.
+	 */
+	public function b_e_a_plugin_option_editable_types( $option ) {
```

Comment:
@SID177 Can you change the name of the function, keep as same as used in rtMedia plugin

Reply:
I've kept the function name same as filter name, will that work?

Reply:
At many places we use same function name as filter name that's why.

Reply:
Can you prefix with rtmedia. don't keep it the same as filter name.

Reply:
Changed function name.